db_graph.php

```diff
@@ -32,7 +32,7 @@
           <div class="input-group-addon">
             <i class="far fa-clock"></i>
           </div>
-          <input type="button" class="form-control pull-right" id="querytime" value="Click to select date and time range">
+          <input type="text" class="form-control pull-right" id="querytime" value="Click to select date and time range" readonly style="background:#fff">
```

Comment:
Can you add a `text-white` class in style.css following the existent classes? This reduces inline styles and duplication

Reply:
`text-white` is misleading because the background color must be set.
What about using `bg-white` class and add
```
.bg-white {
    background-color: #fff !important;
}
```
to `style.css`?
(Just tested and works for me)

Reply:
Yeah, sorry I meant `bg-white`. The above is what I'd do so go ahead :)

Reply:
I believe https://github.com/pi-hole/AdminLTE/pull/1103/commits/98b36d82a46464cc8938216f47c78fb9c2ad39e8 is what you are asking for?

Reply:
Almost :) Can you put `bg-white` before the `text-*` utils and also squash your patches?

Reply:
[Done](https://github.com/pi-hole/AdminLTE/commit/49d94e179a9f565c4dd8465fdffb9e116d8f3b73), thanks